packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -95,6 +95,9 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
 
   const onTexFieldKeyDown = useCallback(
     (ev: React.KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+      if ((ev.nativeEvent as KeyboardEvent).isComposing) {
```

Comment:
Avoid `as` as much as possible as then typescript can't typeguard against misuse.
Looks like the `as` shouldn't be needed?:
![image](https://user-images.githubusercontent.com/2684369/153941372-9774a739-210f-446b-92eb-df9df6431c15.png)